docs/jwst/wfss_contam/arguments.rst

```diff
@@ -24,9 +24,20 @@ The ``wfss_contam`` step uses the following optional arguments.
   fractions refer to the total available cores and on most CPUs these include physical
   and virtual cores.
 
-``--brightest_n``
-  Limits the number of direct image sources that are dispersed to only the brightest n sources.
-  If set to ``None``, all sources will be used. Defaults to ``None``.
+``--orders``
+  A string indicating which grism order(s) to simulate. The default value is None, which
+  means all orders that are defined in the wavelength and specwcs reference files
+  for that instrument will be simulated.
+  To specify a single order, use a string like "0" or "1". To specify multiple orders,
```

Comment:
It's actually expecting a list or there's an error:
ValidationError: Config parameter 'steps/wfss_contam/orders': the value "2" is of the wrong type.

I think a list makes more sense in this application, but this documentation should be updated.